tests/minifier.js

```diff
@@ -535,34 +535,34 @@ test('remove CDATA sections from scripts/styles', function() {
   output = '<script>alert(11)</script>';
   equal(minify(input, { minifyJS: true }), output);
 
-  input = '<style><![CDATA[\np.a{backgourd:red}\n]]></style>';
```

Comment:
It's pretty lightweight and gets the job done, with pretty good hints / auto-completion, integrates well with git and won't get in my way like a full-featured IDE would.

So yeah I've been liking it so far :+1: 

My history of code editors:
- `edit.com`
- Notepad
- Visual Studio
- Eclipse
- NetBeans
- VS Code